src/collectors/prometheus_rabbitmq_queues_collector.erl

```diff
@@ -53,6 +53,7 @@
                          running -> 0;
                          undefined -> undefined;
                          down -> undefined;
+                         flow -> undefined;
```

Comment:
I'm sure "undefined" would be the best value here, but it seemed better than zero or MsgCount.